src/types.ts

```diff
@@ -52,8 +52,14 @@ export interface BidsDto {
 export interface Bid extends BidParams {
   date: number;
   signedMessage: string;
+  version: string
 }
 
+export interface MaybeBid extends BidParams {
```

Comment:
Confusing overlap with Maybe<Bid>

Try and find a name that's contextually makes sense

Reply:
yeah I wasn't sure it felt weird, will adjust